pkg/webhook/webhook.go

```diff
@@ -74,6 +82,35 @@ func AddToManager(mgr manager.Manager, poolManager *pool_manager.PoolManager) er
 	if err != nil {
 		return err
 	}
-
 	return nil
 }
+
+func CreateOwnerRefForMutatingWebhook(kubeClient *kubernetes.Clientset) error {
+	managerStatefulSet, err := kubeClient.AppsV1().StatefulSets("kubemacpool-system").Get("kubemacpool-mac-controller-manager", metav1.GetOptions{})
```

Comment:
Isn't it possible to set the reference while the MutatingWebhookConfiguration is being created? Or is the code hidden somewhere?